tests/square_integration/Notices_Controller_Test.php

```
@@ -226,4 +233,109 @@ public function it_should_render_remotely_disconnected_notice(): void {
+	public function it_should_render_token_invalid_notice(): void {
+		$this->assertMatchesHtmlSnapshot( $this->make_controller()->render_token_invalid_notice() );
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_only_show_token_refresh_failing_notice_after_repeated_failures(): void {
+		$merchant   = tribe( Merchant::class );
+		$controller = $this->make_controller();
+
+		$merchant->update( [ 'expires_at' => gmdate( 'c', time() + HOUR_IN_SECONDS ) ] );
```

PHPStan (2.2.7)

[error] Constant HOUR_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)